lib/osf-components/addon/components/form-controls/wrapper/template.hbs

```diff
@@ -0,0 +1,12 @@
+{{#let (unique-id) as |inputElementId|}}
```

Comment:
Just wondering what the benefit is of creating the wrapper component instead of adding a label/validation errors to the power-select component directly?  At least for the scope of this ticket.

Reply:
This is more for future's sake. I am adding the wrapper component so that we can use the component in the future. We should also convert the `radioGroup` component to use the same thing, but that'll be a separate ticket.